schemas/rum/vital-schema.json

```diff
@@ -42,6 +42,16 @@
               "description": "Name of the vital, as it is also used as facet path for its value, it must contain only letters, digits, or the characters - _ . @ $",
               "readOnly": true
             },
+            "details": {
+              "type": "string",
+              "description": "Details of the vital",
```

Comment:
**Question**: by details you mean a description of what the vital is about ?

Reply:
The field is used to store for example a secondary identifier. Which will be used to track vitals for React component.